### PR TITLE
[hack/en] Fixed some typos and added pt-br translation

### PR DESCRIPTION
**Pull request by davelima**
Hi! I fixed a few typos on hack/en and added a pt-br translation.

### CODE REVIEW

File: pt-br/hack-pt.html.markdown [hunk 1]
```diff
@@ -0,0 +1,316 @@
+---
+language: Hack
+contributors:
+    - ["Stephen Holdaway", "https://github.com/stecman"]
+    - ["David Lima", "https://github.com/davelima"]
+translators:
+    - ["David Lima", "https://github.com/davelima"]
+lang: pt-br
+filename: learnhack.hh
```
Comment (vendethiel) [inline pt-br/hack-pt.html.markdown]: please add a suffix to the filename

Comment (davelima) [inline pt-br/hack-pt.html.markdown]: Done!

**Comment by vendethiel** [comment]
Thank you very much!

**Comment by vendethiel** [comment]
:star:

**Pull request merged**